rebar.config

```diff
@@ -9,9 +9,9 @@
     {platform_define, "^[2-9]", 'POST_OTP_18'}
 ]}.
 
-{project_plugins, [covertool, rebar3_ex_doc, {rebar3_hank, "~> 0.3.0"}]}.
+{project_plugins, [covertool, rebar3_ex_doc, {rebar3_hank, "1.4.0"}]}.
 
-{deps, [{hex_core, "0.8.4"}, {verl, "1.1.1"}]}.
+{deps, [{hex_core, {git, "https://github.com/paulo-ferraz-oliveira/hex_core", {branch, "fix/spec"}}}, {verl, "1.1.1"}]}.
```

Comment:
This is obviously just to show that with this we have ✅  CI. I don't expect to keep this change and/or maintain that branch 😄

Reply:
`hex_core` 0.10.2 is out, so I've imported it here, instead of this workaround.